app/utils/event-deadline-date-finder.ts

```diff
@@ -4,26 +4,57 @@ import moment from 'moment';
 const daysToWaitAfterSubmission = config.get('daysToWait.afterSubmission');
 const daysToWaitAfterReasonsForAppeal = config.get('daysToWait.afterReasonsForAppeal');
 
-function getDeadline(currentAppealStatus: string, history) {
+/**
+ * Finds a targeted direction, retrieves it's due date and returns it as a string with the correct date format
+ * @param directions the directions
+ * @param directionTagToLookFor the direction to find
+ */
+function getFormattedDirectionDueDate(directions: Direction[], directionTagToLookFor: string) {
+  let formattedDeadline = null;
+  const direction = directions.find(d => d.tag === directionTagToLookFor);
+  if (direction) {
+    const dueDate = direction.dueDate;
+    formattedDeadline = moment(dueDate).format('DD MMMM YYYY');
```

Comment:
Can we pull this out to a util class think we are going to be showing dates like this all over the place soon.